integration_tests/test_dict_11.py

```diff
@@ -0,0 +1,8 @@
+from lpython import i32
+
+def test_dict_11():
+    num : dict[i32, i32]
+    num = {11: 22, 33: 44, 55: 66}
+    assert num.get(7, -1) == -1
```

Comment:
Test for the case following case,

```suggestion
    assert num.get(11, -1) == 22
    assert num.get(33, -1) == 44
    assert num.get(55, -1) == 66
    assert num.get(72, -110) == -110
```